css-showcases/src/transition-helper.js

```diff
@@ -1,4 +1,33 @@
-import styles from './transition-box.module.scss';
+const styles = /*css*/ `.transition-box-label {
+  font-family: monospace;
+  margin: 0.125rem;
+  font-size: 0.75rem;
+  color: #fff;
+}
+
+.transition-box-label-sm {
+  font-size: 0.55rem;
+}
+
+.transition-box-wrapper {
+  display: flex;
+  flex-direction: column;
+}
+
+.transition-box {
+  cursor: pointer;
+  height: 6rem;
+  width: 12rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #fff;
+  border-radius: 0.375rem;
+  box-shadow: 0 25px 50px -12px rgba(0, 0, 0, 0.25);
+  background-color: #6366f1;
+}
+`;
```

Comment:
we don't have Shadow DOM in `css-showcases` and we relied on CSS modules for unique style selectors generation

now when you refactored this, we have a higher chance of running in the styles clash, I think this is not an improvement

Reply:
Right. I was trying to move away from css files that need global css import for usage (as it was mentioned in another comment) - but to do so we need shadow dom for encapsulation to avoid global collision... So I'm restoring this to css modules and we'll migrate to lit + shadow dom for this component at a later time.